src/services/user.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { Repository } from "typeorm";
+import { User } from "../entities/user.entity";
+
+@Injectable()
+export class UserService {
+	constructor(
+		@InjectRepository(User)
+		private readonly userRepository: Repository<User>,
+	) {}
+
+	async findAll(): Promise<User[]> {
+		return await this.userRepository.find();
```

Comment:
please add UserMapperService with toDTO() method
that method will transform your entity to a UserDTO interface